org._2006Scape.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org._2006Scape.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk8
+modules:
+- name: openjdk
+  buildsystem: simple
+  build-commands:
+  - "/usr/lib/sdk/openjdk8/install.sh"
+- name: 2006Scape
+  buildsystem: simple
+  build-commands:
+  - install -D run.sh /app/bin/run.sh
+  - install -D 2006Scape_Launcher.jar /app/bin/2006Scape_Launcher.jar
+  - install -D org._2006Scape.Launcher.desktop /app/share/applications/org._2006Scape.Launcher.desktop
+  - install -D icon.svg /app/share/icons/hicolor/scalable/apps/org._2006Scape.Launcher.svg
+  - install -D org._2006Scape.Launcher.appdata.xml /app/share/metainfo/org._2006Scape.Launcher.appdata.xml
+  sources:
+  - type: file
+    path: run.sh
+  - type: file
+    path: 2006Scape_Launcher.jar
```

Comment:
At least this file should come from external source. GitHub is not a hosting for binary files.